cmd/exchange_wrapper_standards/exchange_wrapper_standards_test.go

```diff
@@ -547,6 +547,7 @@ var acceptableErrors = []error{
 	context.DeadlineExceeded,             // If the context deadline is exceeded, it is not an error as only blockedCIExchanges use expired contexts by design
 	order.ErrPairIsEmpty,                 // Is thrown when the empty pair and asset scenario for an order submission is sent in the Validate() function
 	deposit.ErrAddressNotFound,           // Is thrown when an address is not found due to the exchange requiring valid API keys
+	currency.ErrPairNotFound,             // Is thrown when a pair is not found in a pair matching function
```

Comment:
Shizbit, I have a diff for you! I wanted to ensure that your new interface funcs are tested properly and have made small amendments:
https://gist.github.com/gloriousCode/f30179c1ad95c371385075f999f041cc